vasquez_a03b.js

```diff
@@ -0,0 +1,41 @@
+const firstName = "Bryan";
```

Comment:
Your script should start off with a comment block as indicated in the assignment.
-2